### PR TITLE
update: use original `Data` field instead of specific PowerShell Classic log's extracted field

### PR DESCRIPTION
**Pull request by fukusuket**
### Summary of the Pull Request
Changed detection rules in `PowerShell Classic` to refer original `Data` fields instead of extracted specific fields. 
Please see the discussion below for more information on the background behind this PR.
https://github.com/SigmaHQ/sigma/discussions/4510

~~If there are no problems with the changes in this PR, I will also modify the `Task Scheduler` detection rules in another PR.~~
(↑Sorry, the Task scheduler event log fields was defined in XML.)

I would appreciate it if you could review.

### Changelog
update: Suspicious XOR Encoded PowerShell Command Line - PowerShell
update: Uncommon PowerShell Hosts
update: Delete Volume Shadow Copies Via WMI With PowerShell
update: PowerShell Downgrade Attack - PowerShell
update: PowerShell Called from an Executable Version Mismatch
update: Netcat The Powershell Version
update: Remote PowerShell Session (PS Classic)
update: Renamed Powershell Under Powershell Channel
update: Suspicious PowerShell Download
update: Use Get-NetTCPConnection
update: Zip A Folder With PowerShell For Staging In Temp - PowerShell
update: Tamper Windows Defender - PSClassic
update: Suspicious Non PowerShell WSMAN COM Provider
update: Suspicious XOR Encoded PowerShell Command Line - PowerShell

### Example Log Event
The EID:400 log sample for `Windows PowerShell.evtx` is as follows: 
(It's not malicious one. But I'm pasting it so you can see the evtx format specification)
```xml
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="PowerShell" /> 
    <EventID Qualifiers="0">400</EventID> 
    <Version>0</Version> 
    <Level>4</Level> 
    <Task>4</Task> 
    <Opcode>0</Opcode> 
    <Keywords>0x80000000000000</Keywords> 
    <TimeCreated SystemTime="2023-10-09T13:24:55.2952147Z" /> 
    <EventRecordID>1643</EventRecordID> 
    <Correlation /> 
    <Execution ProcessID="4684" ThreadID="0" /> 
    <Channel>Windows PowerShell</Channel> 
    <Computer>HOSTA</Computer> 
    <Security /> 
  </System>
  <EventData>
    <Data>Available</Data> 
    <Data>None</Data> 
    <Data>NewEngineState=Available PreviousEngineState=None SequenceNumber=13 HostName=ConsoleHost HostVersion=5.1.22621.963 HostId=e2a4b19b-92ef-407b-9fe0-b646489b66c5 HostApplication=C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command Get-WmiObject Win32_Process | Where-Object { $_.ExecutablePath -eq 'C:\Users\alice\AppData\Local\Programs\Microsoft VS Code\bin\code-tunnel.exe' } | Select @{Name='Id'; Expression={$_.ProcessId}} | Stop-Process -Force EngineVersion=5.1.22621.963 RunspaceId=e1518a29-e3b8-41d3-931a-ff4db039a628 PipelineId= CommandName= CommandType= ScriptName= CommandPath= CommandLine=</Data> 
  </EventData>
</Event>
```

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/powershell/powershell_classic/posh_pc_abuse_nslookup_with_dns_records.yml [hunk 1]
```diff
@@ -9,23 +9,22 @@ references:
...
-modified: 2022/12/19
+modified: 2023/10/27
 tags:
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_abuse_nslookup_with_dns_records.yml:19]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_alternate_powershell_hosts.yml [hunk 2]
```diff
@@ -9,24 +9,25 @@ references:
...
-modified: 2023/04/12
+modified: 2023/10/27
 tags:
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_alternate_powershell_hosts.yml:19]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_delete_volume_shadow_copies.yml [hunk 3]
```diff
@@ -7,23 +7,22 @@ references:
...
-modified: 2021/10/16
+modified: 2023/10/27
 tags:
     - attack.impact
     - attack.t1490
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_delete_volume_shadow_copies.yml:17]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_downgrade_attack.yml [hunk 4]
```diff
@@ -6,20 +6,20 @@ references:
...
+modified: 2023/10/27
 tags:
     - attack.defense_evasion
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_downgrade_attack.yml:17]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_exe_calling_ps.yml [hunk 5]
```diff
@@ -6,23 +6,24 @@ references:
...
+modified: 2023/10/27
 tags:
     - attack.defense_evasion
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_exe_calling_ps.yml:17]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_powercat.yml [hunk 6]
```diff
@@ -11,19 +11,19 @@ references:
...
-modified: 2022/12/25
+modified: 2023/10/27
 tags:
     - attack.command_and_control
     - attack.t1095
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_powercat.yml:21]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_remote_powershell_session.yml [hunk 7]
```diff
@@ -18,12 +18,13 @@ tags:
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_remote_powershell_session.yml:21]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_renamed_powershell.yml [hunk 8]
```diff
@@ -6,21 +6,21 @@ references:
...
-modified: 2021/10/16
+modified: 2023/10/27
 tags:
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_renamed_powershell.yml:16]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_susp_download.yml [hunk 9]
```diff
@@ -9,21 +9,21 @@ references:
...
-modified: 2022/11/09
+modified: 2023/10/27
 tags:
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_susp_download.yml:19]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_susp_get_nettcpconnection.yml [hunk 10]
```diff
@@ -6,17 +6,17 @@ references:
...
-modified: 2022/12/25
+modified: 2023/10/27
 tags:
     - attack.discovery
     - attack.t1049
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_susp_get_nettcpconnection.yml:16]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_susp_zip_compress.yml [hunk 11]
```diff
@@ -9,22 +9,23 @@ references:
...
-modified: 2022/12/02
+modified: 2023/10/27
 tags:
     - attack.collection
     - attack.t1074.001
 logsource:
     product: windows
     service: powershell-classic
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_susp_zip_compress.yml:19]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_tamper_windows_defender_set_mp.yml [hunk 12]
```diff
@@ -9,19 +9,19 @@ references:
...
-modified: 2023/07/13
+modified: 2023/10/27
 tags:
     - attack.defense_evasion
     - attack.t1562.001
 logsource:
     product: windows
     category: ps_classic_provider_start
-    definition: fields have to be extracted from the event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_tamper_windows_defender_set_mp.yml:19]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_wsman_com_provider_no_powershell.yml [hunk 13]
```diff
@@ -17,12 +17,12 @@ tags:
 logsource:
     product: windows
     service: powershell-classic
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_wsman_com_provider_no_powershell.yml:20]:
```suggestion
```

File: rules/windows/powershell/powershell_classic/posh_pc_xor_commandline.yml [hunk 14]
```diff
@@ -6,22 +6,22 @@ references:
...
-modified: 2022/12/02
+modified: 2023/10/27
 tags:
     - attack.execution
     - attack.t1059.001
 logsource:
     product: windows
     category: ps_classic_start
-    definition: fields have to be extract from event
+    definition: No need to extract fields from the event
```
Comment (nasbench) [inline rules/windows/powershell/powershell_classic/posh_pc_xor_commandline.yml:16]:
```suggestion
```

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Thanks for the PR @fukusuket. Just made a quick update to use the `contains` modifier instead of wildcards (as it doesn't impact the rule that much since all the data that's interesting shouldn't colid with any other field.

Also just FYI, something to take care in hayabusa is that some of the fields are localized. For example `HostApplication` in a french layout becomes `Application hote` :) 

Thanks once again :)

**Comment by fukusuket** [comment]
@nasbench 
I didn't know that fields were localized...😲 (It happened to be HostApplication on Japanese OS) 
Thank you so much for the useful information and review :)  We are also considering the field extraction feature in Hayabusa, so we would like to use it as a reference.

**Review by frack113 — APPROVED** [review]

**Pull request merged**